src/py42/services/auditlogs.py

```diff
@@ -0,0 +1,67 @@
+from py42 import settings
+from py42.services import BaseService
+from py42.services.util import get_all_pages
+from py42.util import convert_timestamp_to_str
+from py42.util import to_list
+
+
+class AuditLogsService(BaseService):
```

Comment:
Can we add a docstring with the swagger docs for this microservice?

Reply:
Given that we found the docs are not right or helpful, is it ok if this is not done?

Reply:
Added.